packages/apptive_grid_core/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
   flutter: ">=1.17.0"
 
 dependencies:
-  collection: ^1.17.1
+  collection: ^1.17.2
   image: ^4.0.17
   intl: ^0.18.0
   http: ^0.13.6
```

Comment:
Same here. Please check all packages to make sure the http dependencies are defined correctly